resources/units/aircraft/FA-18E_super-hornet.yaml

```diff
@@ -0,0 +1,69 @@
+carrier_capable: true
+description:
+  'The F/A-18E Super Hornet is twin engine, supersonic fighter that is flown
+  by a single pilot in a "glass cockpit". It combines extreme maneuverability , a
+  deadly arsenal of weapons, and the ability to operate from an aircraft carrier.
+  Operated by several nations, this multi-role fighter has been instrumental in conflicts
+  from 1999 to today.
+
+
+  The Hornet is equipped with a large suite of sensors that includes a radar, targeting
+  pod, and a helmet mounted sight. In addition to its internal 20mm cannon, the Super Hornet
+  can be armed with a large assortment of unguided bombs and rockets, laser and GPS-guided
+  bombs, air-to-surface missiles of all sorts, and both radar and infrared-guided
+  air-to-air missiles.
+
+
+  The Super Hornet is also known for its extreme, slow-speed maneuverability in a dogfight.
+  Although incredibly deadly, the Super Hornet is also a very easy aircraft to fly.'
+introduced: 1999
+manufacturer: Boeing Defense, Space & Security
+origin: USA
+price: 28
+role: Carrier-based Multirole Fighter
+# DCS default livery for the Hornet is the Blue Angels.
+default_livery: "VFA-34"
+fuel:
```

Comment:
Does it really have identical fuel consumption as the legacy hornet? I find that surprising.

Reply:
fuel consumption removed because I sadly do not have the free time to test the aircraft.

Reply:
Works for me. No data is better than bad data. Most aircraft already have no data anyway.